src/peft/tuners/lora.py

```diff
@@ -573,6 +571,29 @@ def _unload_and_optionally_merge(self, merge=True, progressbar: bool = False):
                         padding=target.padding,
                         dilation=target.dilation,
                     )
+                elif is_bnb_available() and isinstance(target, bnb.nn.Linear8bitLt):
+                    bias = target.bias is not None
+                    new_module = bnb.nn.Linear8bitLt(
+                        target.in_features,
+                        target.out_features,
+                        bias,
```

Comment:
Just a small thing, but you could you please use keyword arguments starting from the `bias` argument, i.e. 

```python
                    new_module = bnb.nn.Linear8bitLt(
                        target.in_features,
                        target.out_features,
                        bias=bias,
                        # etc.
```

Same change below for 4bit. This should be more future proof and conforms with the use of kw arguments in bnb.

Reply:
fixed